### PR TITLE
docs: add release date to component page

### LINKED ISSUE #1030: Read release and some other information from ComponentStatus files and add it to storybook page wrapper
There's lots of information available now in ComponentStatus file. We can import that info object in StoryPageWrapper and add it in header of each story.

### PR DESCRIPTION
**Pull request by saurabhdaware**
Resolves #1030 

Adds release date to all component's story page. This will tell users which minimum version they need to upgrade to for using that component. Saves few clicks for user

image

### CODE REVIEW

File: packages/blade/src/utils/storybook/ComponentStatusTable.tsx [hunk 1]
```diff
@@ -19,408 +21,6 @@ import { Link } from '~components/Link';
...
 
-type ComponentStatuses =
-  | 'released'
-  | 'in-api-spec'
-  | 'in-development'
-  | 'in-design'
-  | 'deprecated'
-  | 'to-be-decided'
-  | `planned-Q${1 | 2 | 3 | 4}-${'dev' | 'design'}`;
-
```
Comment (saurabhdaware) [inline packages/blade/src/utils/storybook/ComponentStatusTable.tsx:30]: Moved the componentData code to separate file

File: packages/blade/src/utils/storybook/componentData.ts [hunk 2]
```diff
@@ -0,0 +1,404 @@
+type ComponentStatuses =
```
Comment (anuraghazra) [inline packages/blade/src/utils/storybook/componentData.ts:1]: rename this file to `componentStatusData.ts`

File: packages/blade/src/utils/storybook/componentData.ts [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+type ComponentStatusData = {
+  name: string;
+  status: ComponentStatuses;
+  description: string;
+  releasedIn?: string;
+  storybookLink?: string;
+}[];
+
+const componentData: ComponentStatusData = [
```
Comment (anuraghazra) [inline packages/blade/src/utils/storybook/componentData.ts:18]: and let's rename this variable also

**Review by anuraghazra — DISMISSED** [review]

**Review by anuraghazra — APPROVED** [review]

**Comment by saurabhdaware** [comment]
merging it on L1 since its a small documentation change

**Pull request merged**